### ISSUE TITLE
Vue preset broken after Prettier 1.13.0 release

### ISSUE THREAD

**Issue by edmorley** [mozilla-neutrino/neutrino-dev#913]
Prettier 1.13.0 made a breaking change to their API, which causes the following error during `yarn start` when using `@neutrinojs/vue`:

```
ERROR in ./node_modules/vue-loader/lib/template-compiler?{"id":"data-v-04c2046b","hasScoped":false,"buble":{"transforms":{}}}!./node_modules/vue-loader/lib/selector.js?type=template&index=0!./src/App.vue
Module build failed: Error: No parser and no file path given, couldn't infer a parser.
    at normalize (C:\Users\Ed\src\vue-test\node_modules\prettier\index.js:7051:13)
    at formatWithCursor (C:\Users\Ed\src\vue-test\node_modules\prettier\index.js:10370:12)
    at C:\Users\Ed\src\vue-test\node_modules\prettier\index.js:31115:15
    at Object.format (C:\Users\Ed\src\vue-test\node_modules\prettier\index.js:31134:12)
    at Object.module.exports (C:\Users\Ed\src\vue-test\node_modules\vue-loader\lib\template-compiler\index.js:80:23)
```

This has been fixed upstream in vuejs/component-compiler-utils#15, however that package is only used by vue-loader v15, and we're using vue-loader v14 on `master` and v13 on `release/v8`, which still have the old behaviour:
https://github.com/vuejs/vue-loader/blob/v13.7.0/lib/template-compiler/index.js#L80
https://github.com/vuejs/vue-loader/blob/v14.2.2/lib/template-compiler/index.js#L73

We can fix on `master` by updating to vue-loader v15, however for `release/v8` our options are:
* update from vue-loader v13 to v15 even though it's a breaking change
* ask if there could be a vue-loader 13.x backport of the fix
* use yarn the resolutions to downgrade the prettier subdependency to 1.12.1 (but this doesn't help npm - though I really don't want to make another temporarily forked package like we did for babel-minify-webpack-plugin)

**Comment by edmorley** [comment]
I've opened PRs against both vue-loader 13 and 14:
vuejs/vue-loader#1322
vuejs/vue-loader#1323

**Comment by azz** [comment]
Looks like it is fixed in `vue-loader@13.7.2` and `vue-loader@14.2.3`.
